core/src/trezor/loop.py

```diff
@@ -126,6 +126,15 @@ def run() -> None:
                 # rationale: We use untyped lists here, because that is what the C API supports.
 
 
+def clear():
+    """Clear all queue state.  Any scheduled or paused tasks will be forgotten."""
+    _ = [0, 0, 0]
+    while _queue:
+        _queue.pop(_)
```

Comment:
Is the utimeq documented anywhere? Is it ours or micropython's?

Reply:
It is micropythons with a minor addition of one method, by us.